### PR TITLE
Add GetClaim() Method

### PR DESCRIPTION
**Pull request by spottedmahn**
Reference #169

### CODE REVIEW

File: src/IdToken.ts [hunk 1]
```diff
@@ -89,4 +90,29 @@ export class IdToken {
...
+  }
+
+  private static getClaimInternal(claimName: string, decodedIdToken: any, logger: Logger) {
+    let res: string = null;
+
+    if (decodedIdToken.hasOwnProperty(claimName)) {
+      res = decodedIdToken[claimName];
+    }
+    else{
+      logger.warning('unable to find claim: ' + claimName);
```
Comment (saraford) [inline src/IdToken.ts]: You might want to handle the case when logger is null.

Comment (spottedmahn) [reply src/IdToken.ts]: absolutely, thanks @saraford !

**Comment by Might-E** [comment]
Are there any expectations when there will be a new release? We really need this for a new feature we are building in our webapp, but don't want to use some dirty hack to get it to remove it again in a few days because this PR has been added :)

**Comment by davidmorissette** [comment]
@spottedmahn : Do you think `getClaim()` should be implemented on the user class instead of UserAgentApplication ?

**Comment by spottedmahn** [comment]
Hi @davidmorissette - that's probably a better approach; agreed.

**Comment by rohitnarula7176** [comment]
@spottedmahn @davidmorissette Thank you for your contribution. Based on our discussion on this issue, I have already exposed the id_token as property on the user object in the dev branch. Closing this pull request.

**Pull request closed without merging**